lib/util.js

```diff
@@ -230,7 +230,7 @@ exports.getVideoID = function(link) {
     var s = parsed.pathname.split('/');
     id = s[s.length - 1];
   }
-  if (!id) {
+  if (!idRegex.test(id)) {
     throw new Error('No video id found: ' + link);
```

Comment:
Problem with this is if user gives a link when they accidentally didn't copy the last character of the id, they will get the error

    No video id found: http://www.youtube.com/watch?v=th75xfvBi5

Which looks like it's a correct link, but it's missing a character. Maybe a better solution is to still check for `if (!id)` and use this message then, and then also check if the id is set but did not match with the regexp, and have some other error for that.

Something like

`'Video id found (' + id + ') does not match expected format (' + idRegex + ')'`